siriuspy/siriuspy/devices/dvf.py

```diff
@@ -360,6 +369,10 @@ def cam_frames_failures(self):
 
     def cmd_reset(self, timeout=None):
         """Reset DVF to a standard configuration."""
+        # camera device reset
+        self['cam1:DeviceReset'] = 1
```

Comment:
I've tested resetting the device in my local setup. You will also need to set `cam1:RESET` to 1. Basically, that PV tells the EPICS driver to reconnect to the camera.

Something I've been thinking: is any CA put done afterwards (for `prop_values`) synchronous? It might take some time (5~10 sec) for the camera to reset and the driver to reconnect to it.

Any CA put sent before the driver reestablishes the connection will fail.

Reply:
deprecated